utils/travis-build-repo-internal.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
Did you check if this is necessary? Probably the one in `utils/travis-build-repo.sh` is enough